documentation/add/10_single_filebacked_operations.md

```diff
@@ -0,0 +1,77 @@
+# Intermediaries of Single Filebacked Operations
+Date: 2023-01-11
+
+## Objectives and problems to solve
+
+Currently the implementation of single-filebacked objects is based upon a temporary file beinhg created as infrequently
+as possible and modifications to the sqw data taking place within that file. The issue with this mechanism is that, while
```

Comment:
This is not what is currently happens. 
Two currently  working ways of creating new sqw files is 
1) Calculate pix in memory and store them in a file
2) `gen_sqw` -- combining pixels according to bins taken from different smaller sqw files
Everything else is subset of these two operations
Third way -- combining final file from modified pages implemented by Tessella is broken.

Reply:
"Current" is the `single-fb` branch